intellij/src/saros/intellij/eventhandler/colorscheme/AnnotationReloader.java

```diff
@@ -0,0 +1,81 @@
+package saros.intellij.eventhandler.colorscheme;
+
+import com.intellij.openapi.editor.colors.EditorColorsListener;
+import com.intellij.openapi.editor.colors.EditorColorsManager;
+import com.intellij.openapi.project.Project;
+import com.intellij.util.messages.MessageBusConnection;
+import org.jetbrains.annotations.NotNull;
+import saros.intellij.editor.annotations.AnnotationManager;
+import saros.intellij.eventhandler.IProjectEventHandler;
+
+/**
+ * Reloads all current annotations when the IDE color scheme changes to ensure that the annotation
+ * colors match the new theme.
+ */
+public class AnnotationReloader implements IProjectEventHandler {
+  private final Project project;
+  private final AnnotationManager annotationManager;
+
+  private MessageBusConnection messageBusConnection;
+  private boolean enabled;
+  private boolean disposed;
+
+  private final EditorColorsListener editorColorsListener = scheme -> reloadAnnotations();
+
+  public AnnotationReloader(Project project, AnnotationManager annotationManager) {
+    this.project = project;
+    this.annotationManager = annotationManager;
+
+    this.enabled = false;
+    this.disposed = false;
+  }
+
+  /**
+   * Reloads all current Saros annotations.
+   *
+   * @see AnnotationManager#reloadAnnotations()
+   */
+  private void reloadAnnotations() {
```

Comment:
Where is this called ? Cannot find any reference in this class.

Reply:
In line 23. Its a bit hidden with the lambda stuff.